docs/sources/datasources/pyroscope/query-profile-data.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Use the query editor to explore your Pyroscope data.
+keywords:
+  - query
+  - profiling
+  - pyroscope
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Query profile data
+menuTitle: Query profile data
+weight: 300
+---
+
+# Query profile data
+
+The Pyroscope data source query editor gives you access to a profile type selector, a label selector, and collapsible options.
+
+![Query editor](/media/docs/pyroscope/query-editor/query-editor.png 'Query editor')
+
+To access the query editor:
+
+1. Sign into Grafana or Grafana Cloud.
+1. Select your Pyroscope data source.
+1. From the menu, choose **Explore**.
+
+1. Select a profile type from the drop-down menu.
+
+   {{< figure src="/media/docs/pyroscope/query-editor/select-profile.png" class="docs-image--no-shadow" max-width="450px" caption="Profile selector" >}}
+
+1. Use the labels selector input to filter by labels. Pyroscope uses similar syntax to Prometheus to filter labels.
+   Refer to [Pyroscope documentation](https://grafana.com/docs/pyroscope/latest/) for available operators and syntax.
+
+   While the label selector can be left empty to query all profiles without filtering by labels, the profile type or app must be selected for the query to be valid.
+
+   Grafana doesn't show any data if the profile type or app isn’t selected when a query runs.
+
+   ![Labels selector](/media/docs/pyroscope/query-editor/labels-selector.png 'Labels selector')
+
+1. Expand the **Options** section to view **Query Type** and **Group by**.
+   ![Options section](/media/docs/pyroscope/query-editor/options-section.png 'Options section')
+
+1. Select a query type to return the profile data which can be shown in the [Flame Graph][flame-graph], metric data visualized in a graph, or both. You can only select both options in a dashboard, because panels allow only one visualization.
```

Comment:
nit: passive voice _can be_

Reply:
```suggestion
1. Select a query type to return the profile data. Data is shown in the [Flame Graph][flame-graph], metric data visualized in a graph, or both. You can only select both options in a dashboard, because panels allow only one visualization.
```